web/themes/custom/cloudy/src/components/_header.scss

```diff
@@ -258,7 +286,8 @@
   // Content region - /front, /search, /search/*, /404
   &--large {
     border-bottom: $search-large-border-style-underline;
-    border-radius: $cloudy-space-0 $cloudy-space-0 $border-radius-sm $border-radius-sm;
+    border-radius: $cloudy-space-0 $cloudy-space-0 $border-radius-sm
+      $border-radius-sm;
```

Comment:
@RichardDavies , do you know why these values are breaking to the next line? Code lint?

Reply:
Hmm, I thought I'd fixed that in my last story. I'm not sure if Brit re-introduced it with her commit or if it is some code linting. I've just pushed up a new commit to restore it to one line.